<launch>
  <arg name="manager" default="feature_matching_detector_manager" />
  <arg name="standalone" default="false" />
  <arg name="features" />
  <arg name="camera_info" />

  <node if="$(arg standalone)" name="$(arg manager)" pkg="nodelet" type="nodelet" args="manager" output="screen" />

  <node name="feature_matching_detector" pkg="nodelet" type="nodelet" args="load object_detection/FeatureMatchingDetector $(arg manager)" output="screen">
    <remap from="feature_matching_detector/features" to="$(arg features)" />
    <remap from="feature_matching_detector/camera_info" to="$(arg camera_info)" />
    <rosparam file="$(find object_detection)/cfg/feature_matching_detector_params.yaml" />
    <param name="db_type" value="filesystem" />
    <param name="connection_string" value="$(find object_detection)/models" />
  </node>
</launch>
